shippable.yml

```diff
@@ -6,29 +6,31 @@ jdk:
 
 env:
   matrix:
-    - PROFILE="pitest-checkstyle-xpath"
-    - PROFILE="pitest-checkstyle-filters"
-    - PROFILE="pitest-checks-javadoc"
-    - PROFILE="pitest-checks-imports"
-    - PROFILE="pitest-checks-metrics"
-    - PROFILE="pitest-checks-regexp"
-    - PROFILE="pitest-checks-sizes"
-    - PROFILE="pitest-checks-whitespace"
-    - PROFILE="pitest-checks-misc"
+    - PROFILE="pitest-checks-annotation"
     - PROFILE="pitest-checks-blocks"
     - PROFILE="pitest-checks-coding"
     - PROFILE="pitest-checks-design"
-    - PROFILE="pitest-checks-annotation"
     - PROFILE="pitest-checks-header"
+    - PROFILE="pitest-checks-imports"
+    - PROFILE="pitest-checks-indentation"
+    - PROFILE="pitest-checks-javadoc"
+    - PROFILE="pitest-checks-metrics"
+    - PROFILE="pitest-checks-misc"
     - PROFILE="pitest-checks-modifier"
     - PROFILE="pitest-checks-naming"
-    - PROFILE="pitest-checks-indentation"
-    - PROFILE="pitest-checkstyle-tree-walker"
+    - PROFILE="pitest-checks-regexp"
+    - PROFILE="pitest-checks-sizes"
+    - PROFILE="pitest-checks-whitespace"
+    - PROFILE="pitest-checkstyle-ant"
+    - PROFILE="pitest-checkstyle-api"
     - PROFILE="pitest-checkstyle-common"
+    - PROFILE="pitest-checkstyle-filters"
+    - PROFILE="pitest-checkstyle-gui"
     - PROFILE="pitest-checkstyle-main"
-    - PROFILE="pitest-checkstyle-api"
+    - PROFILE="pitest-checkstyle-packagenamesloader"
+    - PROFILE="pitest-checkstyle-tree-walker"
     - PROFILE="pitest-checkstyle-utils"
-    - PROFILE="pitest-checkstyle-gui"
+    - PROFILE="pitest-checkstyle-xpath"
 
```

Comment:
Please restore ordering of profiles in this file, ordering was used to make validation of existing active big projects first, xpath, filters, javadoc, ..... . Pitest runs too long, and such ordering works.

Reply:
ok, 5 sec

Reply:
done